build-script.sh

```diff
@@ -44,9 +44,11 @@ for dir in $DIRS; do
     V1_10="1.10"
     V1_11="1.11"
     V1_12="1.12"
+    V1_13="1.13"
+    V1_14="1.14"
     if [[ "$VERSION" =~ .*"$V1_7".* || "$VERSION" =~ .*"$V1_8".* ]]; then
       echo "Not installing GRPC for old versions"
-    elif [[ "$VERSION" =~ .*"$V1_9" || "$VERSION" =~ .*"$V1_10" || "$VERSION" =~ .*"$V1_11" || "$VERSION" =~ .*"$V1_12" ]]; then
+    elif [[ "$VERSION" =~ .*"$V1_9" || "$VERSION" =~ .*"$V1_10" || "$VERSION" =~ .*"$V1_11" || "$VERSION" =~ .*"$V1_12" ||  "$VERSION" =~ .*"$V1_13" || "$VERSION" =~ .*"$V1_14" ]]; then
```

Comment:
Didn't already make these changes in PR #445?
https://github.com/newrelic/go-agent/pull/445/files#diff-dc62e2d264855761aa63775271765f676dcd1860f6329ccd38398ca97e8adc42R60-R64